frontend/src/metabase/writeback/components/ActionViz/ActionForm.tsx

```diff
@@ -21,15 +22,19 @@ import {
 interface ActionFormProps {
   onSubmit: OnSubmitActionForm;
   dashcard: ActionDashboardCard;
+  dashboard: Dashboard;
```

Comment:
I think it'd be better to use `page` / `dataAppPage` instead of `dashboard` in context of actions. WDYT?